cogs/utils/errors.py

```diff
@@ -0,0 +1,42 @@
+from discord.ext import commands
+
+
+class PostException(Exception):
+    """
+    Raised when there's a error in the post()
+    wrapper
+    """
+
+    pass
+
+
+class ServiceError(commands.CommandInvokeError):
+    """
+    Subclass of commands.CommandInvokeError.
+    Raised whenever a request to a service
+    returns a failure of some sort.
+    """
+
+    pass
+
+
+class NSFWException(commands.CheckFailure):
+    """
+    Subclass of commands.CheckFailure.
+    Raised whenever a NSFW command is not
+    executed in a NSFW channel.
+    """
```

Comment:
Correctly format docsting, there is no need to mention it's a subclass, that's fairly obvious.
```suggestion
    """Raised whenever a NSFW command is not executed in a NSFW channel."""
```